### PR TITLE
[Partner Center Connector] Added new fields and headers for update api

### PR DESCRIPTION
**Pull request by didalal**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: connectors/PartnerCenterReferrals/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1097 @@
...
+            "in": "header",
+            "name": "MS-CorrelationId",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Correlation Identifier",
+            "x-ms-visibility": "important"
+          },
+          {
+            "default": "60858c1b-2062-40ad-b42c-996b6b5b047d",
+            "description": "Optional, ID to identify the caller - PowerAutomate",
```
Comment (sriyen-msft) [inline connectors/PartnerCenterReferrals/apiDefinition.swagger.json:779]: @didalal Please add a space between Power and Automate like below:

` "description": "Optional, ID to identify the caller - Power Automate",`

File: connectors/PartnerCenterReferrals/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1097 @@
...
```
Comment (sriyen-msft) [inline connectors/PartnerCenterReferrals/apiDefinition.swagger.json:865]: @didalal Please add a space between Power and Automate like below:

`"description": "Optional, ID to identify the caller - Power Automate",`

File: connectors/PartnerCenterReferrals/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1097 @@
...
```
Comment (sriyen-msft) [inline connectors/PartnerCenterReferrals/apiDefinition.swagger.json:940]: @didalal Please add a space between Power and Automate like below:

`"description": "Optional, ID to identify the caller - Power Automate",`

File: connectors/PartnerCenterReferrals/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1097 @@
...
```
Comment (sriyen-msft) [inline connectors/PartnerCenterReferrals/apiDefinition.swagger.json:1031]: @didalal Please add a space between Power and Automate like below:

`"description": "Optional, ID to identify the caller - Power Automate",`

**Comment by didalal** [comment]
> PartnerCenter

File: connectors/PartnerCenterReferrals/apiDefinition.swagger.json (hunk 1 above)
Comment (didalal) [reply connectors/PartnerCenterReferrals/apiDefinition.swagger.json]: Done

**Comment by dougelliott** [comment]
> Could the file path be changed from `connectors/PartnerCenter...` to `certified-connectors/PartnerCenter...`?

@didalal It looks like the file path wasn't updated in your latest commit. Could that be changed please? Thanks!

**Comment by didalal** [comment]
@dougelliott when I fork the repo, I only see connectors folder not certified-connectors

**Comment by dougelliott** [comment]
> @dougelliott when I fork the repo, I only see connectors folder not certified-connectors

@didalal You should be able to manually change the folder name to `certified-connectors` in your commit, but please let me know if you run into any issues with that

**Comment by mamurshe** [comment]
@didalal Please fix the conflict and submit again.

**Comment by sriyen-msft** [comment]
@didalal Please resolve the conflicts and merge the PR

**Comment by didalal** [comment]
@mamurshe @sriyen-msft abandoning this PR and opened created a new PR: https://github.com/microsoft/PowerPlatformConnectors/pull/242

**Pull request closed without merging**